redis/redis.test.w

```diff
@@ -0,0 +1,21 @@
+bring expect;
+bring "./redis.w" as r;
+
+let redis = new r.Redis();
+
+test "set and get" {
```

Comment:
But, when I'm starting this test, it is removing the previous container of port 32719, and starting a new container with port 32720.

Reply:
indeed, in wing tests are isolated. so every time a test is executed, the entire preflight environment is created, so you know you have a clean setup.

(think of preflight as the "setup" of each test in traditional testing frameworks)